Services/IotHub/RegistryManagerWrapper.cs

```diff
@@ -17,30 +17,17 @@ namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.IotHub
     public interface IRegistryManager
     {
         void Init(string connString);
-
+        void ValidateConnectionString(string connString);
```

Comment:
new line before and after the prototype

Reply:
I prefer this compact way given that there are no comments, to keep the file more compact